plugins/cisco_meraki.yaml

```diff
@@ -0,0 +1,30 @@
+# Plugin Info
+version: 0.0.1
+title: Cisco Meraki
+description: Log parser for Cisco Meraki
+parameters:
+  listen_address:
+    label: Listen Address
+    description: A syslog address of the form `<ip>:<port>`
+    type: string
+    default: ":5140"
+
+# Set Defaults
+{{$listen_address := default ":5140" .listen_address}}
+
+# Pipeline Template
+pipeline:
+  - id: meraki_input
+    type: udp_input
+    listen_address: {{ $listen_address }}
+    labels:
+      log_type: cisco_meraki
+    output: meraki_parser
+
+  - id: meraki_parser
+    type: regex_parser
+    regex: '^(?P<timestamp>[a-zA-z]+ \d{2} \d{2}:\d{2}:\d{2}) (?P<message>.*)'
```

Comment:
Does meraki not send syslog messages?

Reply:
I can't definitely answer that. Based on the sample logs MC saved and her fluentd configuration, it doesn't look like it, because they don't start with a priority code and they have custom parsing going on. But then again, I don't know if that's because they saved the sample logs incorrectly.